yt_dlp/extractor/taptap.py

```diff
@@ -0,0 +1,209 @@
+import random
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    int_or_none,
+    join_nonempty,
+    traverse_obj,
+    url_or_none,
+)
```

Comment:
```suggestion
import re
import uuid

from .common import InfoExtractor
from ..utils import (
    clean_html,
    int_or_none,
    join_nonempty,
    str_or_none,
    url_or_none,
)
from ..utils.traversal import traverse_obj
```

Reply:
Is there any reason to import `traverse_obj` separately?

Reply:
`utils.traversal` is preferred since `traverse_obj` is actually defined there, but either works really. You will see both used in existing extractors.